README.md

```diff
@@ -39,7 +39,7 @@ If you  are **not** using the [Apple Search Ads transformation package](https://
 ```yaml
 packages:
   - package: fivetran/apple_search_ads_source
-    version: [">=0.4.0", "<0.5.0"] # we recommend using ranges to capture non-breaking changes automatically
+    version: [">=0.5.0", "<0.6.0"] # we recommend using ranges to capture non-breaking changes automatically
```

Comment:
Also I think we'll need to update this [section](https://github.com/fivetran/dbt_apple_search_ads_source?tab=readme-ov-file#passing-through-additional-metrics) of the README with the new metrics we're including

Reply:
good catch!